RNSound/RNSound.m

```diff
@@ -47,6 +47,9 @@ -(void) audioPlayerDidFinishPlaying:(AVAudioPlayer*)player
   if (key == nil) return;
 
   @synchronized(key) {
+    if (!flag) {
+      [self setOnPlay:flag forKey:key];
```

Comment:
@sunjae1294 This method is called when the audio player finished playing. Thus, `[self setOnPlay:NO forKey:key];` should be called always, isn't it?

Reply:
Actually Yes!! I think changing it to `[self setOnPlay:NO forKey:key];` would be better too